apps/opik-backend/src/main/java/com/comet/opik/domain/TraceDAO.java

```diff
@@ -232,6 +235,16 @@ AND id in (
             ;
             """;
 
+    private static final String TRACE_CNT_BY_WORKSPACE_ID = """
+                SELECT
+                     workspace_id,
+                     COUNT(1) as traces_count
+                 FROM traces
+                 WHERE workspace_id IN :workspace_ids
+                 GROUP BY workspace_id
```

Comment:
@BorisTkachenko i think we need monthly traces ? so query should accept start and end ?

Reply:
@Nimrod007 I thought that we charge for the total amount of traces stored in Clickhouse per organizaion (similar to S3 storage) regardless of the time when they were created. If I'm wrong, please update me with proper requirements.

Reply:
we calculate 2 things :
1) trace count per month (similar to hours per month)
2) trace size (aggregate) which goes to org storage size 

see notion pricing https://www.notion.so/cometml/Pricing-plans-2a81e335e13947699a70e37c5cf0cc50

Reply:
@Nimrod007 Updated endpoint as discussed. Now it returns traces cnt for the previous day for all workspaces. It will be saved on EM side with a daily cron job to be used for Stripe update, blockings based on usage and billing details returned to UI.

Reply:
I believe going with startDate and endDate and using between would improve the index-range operation

Reply:
@thiagohora updated